library/src/scala/tasty/reflect/StandardDefinitions.scala

```diff
@@ -0,0 +1,208 @@
+    def BooleanClass: ClassDef
+
+    /** The class symbol of class `scala.String`. */
+    def StringClass : ClassDef
+
+    /** The class symbol of class `java.lang.Class`. */
+    def ClassClass  : ClassDef
+
+    /** The class symbol of class `scala.Array`. */
+    def ArrayClass  : ClassDef
+
+    /** The module symbol of module `scala.Predef`. */
+    def PredefModule: ValDef
+
+    /** The module symbol of package `java.lang`. */
+    def JavaLangPackage: PackageDef
+
+    /** The module symbol of module `scala.Array`. */
+    def ArrayModule: ValDef
+
+    /** The method symbol of method `apply` in class `scala.Array`. */
+    def Array_apply: DefDef
+
+    /** The method symbol of method `clone` in class `scala.Array`. */
+    def Array_clone: DefDef
+
+    /** The method symbol of method `length` in class `scala.Array`. */
+    def Array_length: DefDef
+
+    /** The method symbol of method `update` in class `scala.Array`. */
+    def Array_update: DefDef
+
+    /** A dummy class symbol that is used to indicate repeated parameters
+      *  compiled by the Scala compiler.
+      */
+    def RepeatedParamClass: ClassDef
+
+    /** The class symbol of class `scala.Option`. */
+    def OptionClass: ClassDef
+
+    /** The module symbol of module `scala.None`. */
+    def NoneModule: ValDef
+
+    /** The module symbol of module `scala.Some`. */
+    def SomeModule: ValDef
+
+    /** Function-like object that maps arity to symbols for classes `scala.Product` */
+    def ProductClass: ClassDef
+
+    /** Function-like object that maps arity to symbols for classes `scala.FunctionX`.
+      *   -  0th element is `Function0`
+      *   -  1st element is `Function1`
+      *   -  ...
+      *   -  Nth element is `FunctionN`
+      */
+    def FunctionClass(arity: Int, isImplicit: Boolean = false, isErased: Boolean = false): ClassDef
+
+    /** Function-like object that maps arity to symbols for classes `scala.TupleX`.
+      *   -  0th element is `NoSymbol`
+      *   -  1st element is `NoSymbol`
+      *   -  2st element is `Tuple2`
+      *   -  ...
+      *   - 22nd element is `Tuple22`
+      *   - 23nd element is `NoSymbol`  // TODO update when we will have more tuples
+      *   - ...
+      */
+    def TupleClass(arity: Int): ClassDef
+
+    /** Contains Scala primitive value classes:
+      *   - Byte
+      *   - Short
+      *   - Int
+      *   - Long
+      *   - Float
+      *   - Double
+      *   - Char
+      *   - Boolean
+      *   - Unit
+      */
+    def ScalaPrimitiveValueClasses: List[ClassDef]
+
+    /** Contains Scala numeric value classes:
+      *   - Byte
+      *   - Short
+      *   - Int
+      *   - Long
+      *   - Float
+      *   - Double
+      *   - Char
+      */
+    def ScalaNumericValueClasses: List[ClassDef]
+  }
+
+  /** Defines standard types.
+    *  @group Definitions
+    */
+  trait StandardTypes {
+    /** The type of primitive type `Unit`. */
+    def UnitTpe: Type
```

Comment:
For readability, I prefer `UnitType` instead of `UnitTpe`. The same for the other names.

Reply:
Changed